assets/stylesheets/_components.scss

```diff
@@ -7,504 +7,504 @@
 // stylesheet order does not matter.
 // ==========================================================================
 
-@import 'layout/style';
+@import '../../client/layout/style.scss';
```

Comment:
I think this specificity is not necessary (prepending `../../client`). We already have special module resolve rules in our default webpack config, so all of these imports should be resolved properly anyway.

Reply:
I hope we can make it possible without. did you try building without those prefixes?

Reply:
I haven't, but I really don't see a good reason to change them considering we don't change our module resolving rules. It's worth to try building without them too.

Reply:
just saying…I tried 😉 

it would be great to update our SASS import paths so this isn't necessary. I think we have to figure this out with the SASS loader though. so far, this was the one out of twenty ways I played with that worked. I'd gladly welcome better solutions!